frontend/src/components/nodes/Nodes.tsx

```diff
@@ -10,13 +10,30 @@ import NodeRow from "./NodeRow";
 
 import { OuterProps } from "../accounts/Accounts";
 
-export default class extends React.Component<OuterProps> {
+interface State {
+  validator: boolean;
```

Comment:
Naming nit: Validator is a role. It is odd to see the type of `validator` being `boolean`.